specs/kubernetes/list_pod_images_by_app.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: List images by app label of pods
+command: >
+  kubectl get pods -l app={{app}} -o jsonpath='{range .items[*]}{"\n"}{range .spec.containers[*]}{.name}{"\t"}{.image}{"\n"}{end}{end}' | sort | uniq
+tags:
+  - kubernetes
+description: Lists the images of pods fetched by app label
+arguments:
+  - name: app
+    description: The value of the app label
+    default_value: ~
+source_url: "https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/"
```

Comment:
Is there a better source of documentation we could link to here instead? This URL doesn't give much indication about what the command does.

Reply:
I switched to the url that documents how to use the `JSONPath`. Let me know if this is not relevant enough.